app/templates/partials/forms/select.html

```diff
@@ -1,12 +1,17 @@
 <select class='input input--select' id='input-{{select.name}}' name='{{select.name}}'>
+
+  {% if not (select.selected and select.mandatory) %}
+      <option value="">{{select.placeholder}}</option>
+  {% endif %}
+
```

Comment:
Hamish is happy. Non-mandatory always has a placeholder regardless if content is present or not. Mandatory has a place holder up until something has been entered then it gets removed